project-base/app/tests/FrontendApiBundle/Functional/Order/GetOrderItemsTest.php

```diff
@@ -48,10 +48,10 @@ public function testGetOrderItems(
     public static function getOrderItemsDataProvider(): iterable
     {
         // first 4 order items
-        yield [['first' => 4], [1, 2, 3, 4]];
+        yield [['last' => 4], [26, 25, 24, 23]];
```

Comment:
I do not understand, why is the filter changed? Btw, the comments should be changed accordingly (or removed :slightly_smiling_face: )

Reply:
Previously, the order items were missing proper sorting. Now that they are sorted by createdAt and id in descending order, I adjusted the pagination from first to last to prevent unnecessary test failures when new order items are added to the data fixtures for this user.

Reply:
ah, I see, thank you!